src/components/Dialog/types.ts

```diff
@@ -22,7 +22,7 @@ export type DialogControlRefProps = {
 export type DialogControlProps = DialogControlRefProps & {
   id: string
   ref: React.RefObject<DialogControlRefProps>
-  isOpen: boolean
+  isOpen?: boolean
```

Comment:
This right here makes it optional, so we can use it in the menu but not need it for the dialog. The way the menu works on web doesn't cause re-renders either like we saw with the dialogs, as it doesn't need to change a prop on the root `<View>`